RocketCastUITests/RocketCastUITests.swift

```diff
@@ -28,9 +30,36 @@ class RocketCastUITests: XCTestCase {
         super.tearDown()
     }
     
-    func testExample() {
-        // Use recording to get started writing UI tests.
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    
+    func testSegueToViews() {
+        let startScreen = self.app.staticTexts["Podcast"]
```

Comment:
I think means you're looking for a "Podcast" text in any of the scenes?

Does it make sense to put the string into a constants file?

I'm not sure myself. Just probing